scripts/create-clientconfig.sh

```diff
@@ -63,7 +63,6 @@ while true; do
     --serviceaccount ) service_account="$2"; shift 2 ;;
     --output-dir ) output_dir="$2"; shift 2 ;;
     --in-cluster-kubeconfig ) in_cluster_kubeconfig="$2"; shift 2 ;;
-    --dev ) dev_mode=true; shift 1 ;;
```

Comment:
So dev mode never really existed?

Reply:
When I was working on the install steps for the docs, I realized that there were some problems with the script if you try to use it with our kind cluster setup. When I wrote it I was really only thinking about real clusters. I wanted to come up with a quick, minimally invasive fix. Initially I thought about a dev mode but abandoned the idea. I missed removing that part previously.